### ISSUE TITLE
1.7.0 release wrong version

### ISSUE THREAD

**Issue by lem0na** [o2sh/onefetch#146]
1.7.0 shows wrong version - 1.6.5
probably due this configuration: https://github.com/o2sh/onefetch/blob/a87ad8b1df9216f100bfd4de831ab68a8d334538/default.nix#L16

**Comment by erikgaal** [comment]
Hi @lem0na,

Can you provide more information please? What platform, which installation method? I am not able to reproduce your issue.

```sh
~ wget https://github.com/o2sh/onefetch/archive/1.7.0.tar.gz -O - | tar zx
~ cd onefetch-1.7.0 && make
# Compiling onefetch v1.7.0
~ target/release/onefetch -V
# onefetch 1.7.0
```

**Comment by spenserblack** [comment]
@erikgaal, I think what happened is that onefetch at tag 1.7.0 \(not v1.7.0\) has the [wrong version in `Cargo.toml`](https://github.com/o2sh/onefetch/blob/1.7.0/Cargo.toml). The executable\(s\) for onefetch 1.7.0 were probably compiled at this tag, which would result in 1.6.5 being reported as the version.

If this is the case, those binaries may need to be re-compiled and re-uploaded.

@lem0na, do you have `cargo` installed? If so, the version available via `cargo install --version 1.7.0 [-f] onefetch` should *definitely* report the correct version.
